src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventRepository.java

```diff
@@ -2,7 +2,7 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.CaseDetails;
```

Comment:
if we are not removing the model package, then better to leave CaseDetails in `uk.gov.hmcts.ccd.domain.model` where all the other domain classes are

Reply:
reverted this to model.definition after discussion